src/fcm.cpp

```diff
@@ -57,7 +57,8 @@ offset fcm::dfdt(const Fragment& fragment, const FCM_params& params, const FCM_s
             result.dr = drdt_chainreaction(0, 1, fragment.radius(), fragment.mass(), result.dm,
                                         params.cloud_disp_coeff, fragment.density());
         } else {
-            result.dr = result.d2r = 0;
+	    result.d2r = 0;
+	    result.dr = drdt_ablation(fragment.radius(), fragment.mass(), result.dm);
```

Comment:
This now means that the if clause does exactly the same thing as the else clause. I like using `drdt_ablation()` here instead of `drdt_chainreaction`.

⇨ I let's make it a parameter in the of the FCM_settings struct whether to use dr/dt = `drdt_ablation()` or to use dr/dt = 0. Then the equivalent class in python must be modified. I'll make that change tomorrow.